core/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties

```diff
@@ -63,6 +63,8 @@ IncompatibleValueType=Values passed to {0} operator must have compatible types
 IncompatibleTypesInList=Values in expression list must have compatible types
 IncompatibleCharset=Cannot apply {0} to the two different charsets {1} and {2}
 InvalidOrderByPos=ORDER BY is only allowed on top-level SELECT
+RecursiveWithMustHaveUnionSetOp=A recursive query only supports UNION [ALL] operator
```

Comment:
Would it be possible to have unit tests forcing to have these two error messages?

Reply:
These errors are protected by "Object not found", currently it is very hard to hit these errors but I thought it is good to protect code path in regards to any future changes with scope etc.

Please let me know if you think these are redundant, then I can remove them.

Reply:
ok, np